docs/noise.ipynb

```diff
@@ -62,196 +62,673 @@
     "\n",
-    "This formalism models evolution of the\n",
-    "density matrix via:\n",
+    "This formalism models evolution of the density matrix $\\rho$ via\n",
     "\n",
-    "$$\\rho \\rightarrow \\sum_k A_k \\rho A_k^\\dagger$$\n",
+    "$$\n",
+    "\\rho \\rightarrow \\sum_{k = 1}^{m} A_k \\rho A_k^\\dagger\n",
+    "$$\n",
     "\n",
-    "Where A<sub>k</sub> are *Kraus* operators. These operators are not\n",
-    "necessarily unitary and must satisfy the trace-preserving property:\n",
+    "where $A_k$ are known as *Kraus operators*. These operators are not necessarily unitary and must satisfy the trace-preserving property\n",
     "\n",
-    "$$\\sum_k A_k^\\dagger A_k = I$$\n",
+    "$$\n",
+    "\\sum_k A_k^\\dagger A_k = I .\n",
+    "$$\n",
     "\n",
-    "As a noisy channel, Kraus operators are not unique. For more details of these operators, see [John Preskill's notes](http://www.theory.caltech.edu/people/preskill/ph219/chap3_15.pdf).\n"
+    "A channel with $m = 1$ unitary Kraus operator is called *coherent*, otherwise the channel is called *incoherent*. For a given noisy channel, Kraus operators are not necessarily unique. For more details on these operators, see [John Preskill's lecture notes](http://theory.caltech.edu/~preskill/ph219/chap3_15.pdf)."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "0e020859769e"
+    "id": "b989869680d4"
    },
    "source": [
-    "## Magic methods\n",
-    "\n",
-    "A `Gate` can represent an operator sum representation by supporting the\n",
-    "`channel` protocol.  Alternatively, for channels that represent probabilistic\n",
-    "mixtures of unitaries, one can implement the `mixture` protocol."
+    "## Common channels"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "c0db375eb631"
+    "id": "708b5b720a74"
    },
    "source": [
-    "### `cirq.channel` and `def _channel_`\n",
-    "\n",
-    "To represent an operator sum evolution, a `Gate` should implement the\n",
-    "`SupportsChannel` protocol. To do this, the `Gate` should implement the\n",
-    "`_channel_(self) -> Sequence[np.ndarray]:` method. \n",
-    "\n",
-    "This method should return the sequence of `numpy` matrices corresponding to the Kraus operators. The basis in which this matrix is expressed is always implicit with respect to the object being called. \n",
-    "\n",
-    "For example, in `GateOperations`, these matrices must be ordered with respect to the list of qubits that the channel is applied to. The qubit-to-amplitude order mapping matches the ordering of `numpy.kron(A, B)`, where `A` is a qubit earlier in the list than the qubit `B`.\n",
+    "Cirq defines many commonly used quantum channels in [`ops/common_channels.py`](https://github.com/quantumlib/Cirq/blob/master/cirq/ops/common_channels.py). For example, we can get a single-qubit bit-flip channel defined as\n",
```

Comment:
Google dev docs style prefers "you" instead of "we": https://developers.google.com/style/highlights
But I would just remove it, if possible, or clarify who you are referring to. This could be "For example, a single-qubit bit-flip ..."

Here and throughout.